chore(docs): install GWAPI from experimental channel (#5193)
Kong/kubernetes-ingress-controller: README.md

You can use [Minikube or Kind][k8s-io-tools] on your local machine or use
a hosted Kubernetes service like [GKE](https://cloud.google.com/kubernetes-engine/).

### Install the Gateway API CRDs

> This command will install all resources that have graduated to beta,
> including GatewayClass, Gateway, ReferenceGrant, and HTTPRoute.

Reviewer: ```suggestion
This command will install all the GA resources,
including `GatewayClass`, `Gateway`, `ReferenceGrant`, and `HTTPRoute`.
```

Author: Thanks! But the `ReferenceGrant` still in v1beta1
https://github.com/kubernetes-sigs/gateway-api/blob/52c2994ed9de1c287a37465490b91cfcf01bf16e/apis/v1beta1/referencegrant_types.go#L43

Author: ```suggestion
This command will install all resources that have graduated to beta,
including `GatewayClass`, `Gateway`, `ReferenceGrant`, and `HTTPRoute`.
```

Author: I referred to the description here https://github.com/kubernetes-sigs/gateway-api/blob/52c2994ed9de1c287a37465490b91cfcf01bf16e/site-src/guides/index.md?plain=1#L38-L40

Reviewer 2: How about "Resources graduated to GA or beta" here?

Reviewer: It looks like that upstream description is a bit out of date. At the moment, Gateway, GatewayClass and HTTPRoute are v1, while ReferenceGrant is beta. +1 on Tao's suggestion.

Reviewer: https://github.com/kubernetes-sigs/gateway-api/pull/2606

```shell
kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
```

> Or, if you want to try some experimental resources and fields such as TCPRoute and UDPRoute,

Reviewer: ```suggestion
Or, if you want to use experimental resources and fields such as `TCPRoute`s and `UDPRoute`s,
```
